fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsFlightPhases.ts

```diff
@@ -0,0 +1,519 @@
+    if (this.previousTargetAltitude !== targetAltitude) {
+      this.previousTargetAltitude = targetAltitude;
+      this._wasBellowThreshold = false;
+      this._wasAboveThreshold = false;
+      this._wasInRange = false;
+      this._wasReach200ft = false;
+      this.fws.soundManager.dequeueSound('cChordOnce');
+      this.fws.soundManager.dequeueSound('cChordCont');
+      return;
+    }
+
+    // Exit when:
+    // - Landing gear down & slats extended
+    // - Glide slope captured
+    // - Landing locked down
+
+    const landingGearIsDown =
+      SimVar.GetSimVarValue('L:A32NX_FLAPS_HANDLE_INDEX', 'Enum') >= 1 &&
+      SimVar.GetSimVarValue('L:A32NX_GEAR_HANDLE_POSITION', 'Percent over 100') > 0.5;
+    const verticalMode = SimVar.GetSimVarValue('L:A32NX_FMA_VERTICAL_MODE', 'Number');
+    const glideSlopeCaptured = verticalMode >= 30 && verticalMode <= 34;
+    const landingGearIsLockedDown = SimVar.GetSimVarValue('GEAR POSITION:0', 'Enum') > 0.9;
+    const isTcasResolutionAdvisoryActive = SimVar.GetSimVarValue('L:A32NX_TCAS_STATE', 'Enum') > 1;
+    if (landingGearIsDown || glideSlopeCaptured || landingGearIsLockedDown || isTcasResolutionAdvisoryActive) {
+      this._wasBellowThreshold = false;
+      this._wasAboveThreshold = false;
+      this._wasInRange = false;
+      this._wasReach200ft = false;
+      this.fws.soundManager.dequeueSound('cChordOnce');
+      this.fws.soundManager.dequeueSound('cChordCont');
+      return;
+    }
+
+    // FIXME better altitude selection
+    const adrAltitude = SimVar.GetSimVarValue(
+      `L:A32NX_ADIRS_ADR_1_BARO_CORRECTED_ALTITUDE_${this.fws.fwsNumber}`,
+      SimVarValueType.Number,
+    );
+    this.fws.fwsNumber;
+    const delta = Math.abs(adrAltitude - targetAltitude);
+
+    if (delta < 200) {
+      this._wasBellowThreshold = true;
+      this._wasAboveThreshold = false;
+      this._wasReach200ft = true;
+    }
+    if (delta > 750) {
+      this._wasAboveThreshold = true;
+      this._wasBellowThreshold = false;
+    }
+    if (delta >= 200 && delta <= 750) {
+      this._wasInRange = true;
+    }
+
+    if (this._wasBellowThreshold && this._wasReach200ft) {
+      if (delta >= 200) {
+        this.fws.soundManager.enqueueSound('cChordCont');
+      } else if (delta < 200) {
+        this.fws.soundManager.dequeueSound('cChordCont');
+      }
+    } else if (this._wasAboveThreshold && delta <= 750 && !this._wasReach200ft) {
+      if (
+        !SimVar.GetSimVarValue('L:A32NX_AUTOPILOT_1_ACTIVE', 'Bool') &&
+        !SimVar.GetSimVarValue('L:A32NX_AUTOPILOT_2_ACTIVE', 'Bool')
+      ) {
+        this.fws.soundManager.dequeueSound('cChordCont');
+        this.fws.soundManager.enqueueSound('cChordOnce');
+      }
+    } else if (delta > 750 && this._wasInRange && !this._wasReach200ft) {
+      if (delta > 750) {
+        this.fws.soundManager.enqueueSound('cChordCont');
+      } else if (delta >= 750) {
+        this.fws.soundManager.dequeueSound('cChordCont');
+      }
```

Comment:
(delta > 750) and (delta >= 750) feels like there's a mistake in the signage here, the 2nd condition seems like dead code here?

Reply:
fixed (hopefully, I hope that's how it was intended)